docs/modules/ROOT/pages/install.adoc

```diff
@@ -31,54 +28,163 @@ These instructions assume all dependencies are installed in the `third-party` di
+| ✅
+| ❌
+| 🐞 Debug
+
+| 🪟 https://mrdox.com/llvm+clang/Windows-RelWithDebInfo-29b20829.7z[`Windows-RelWithDebInfo-29b20829.7z`]
+| 🕵️‍♂️ RelWithDebInfo
+| 🕵️‍♂️ RelWithDebInfo
+| ✅
+| ✅
+| 🚀 Release
+
+| 🪟 https://mrdox.com/llvm+clang/Windows-DebWithOpt-29b20829.7z[`Windows-DebWithOpt-29b20829.7z`]
+| 🔬 DebWithOpt
+| 🐞 Debug
+| ✅
+| ✅
+| 🐞 Debug
+
+| 🐧 https://mrdox.com/llvm+clang/Linux-Release-29b20829.tar.xz[`Linux-Release-29b20829.tar.xz`]
+| 🚀 Release
+| 🚀 Release
+| ❌
+| ✅
+| N/A
+
+| 🐧 https://mrdox.com/llvm+clang/Linux-Debug-29b20829.tar.xz[`Linux-Debug-29b20829.tar.xz`]
+| 🐞 Debug
+| 🐞 Debug
+| ✅
+| ❌
+| N/A
+
+| 🐧 https://mrdox.com/llvm+clang/Linux-RelWithDebInfo-29b20829.tar.xz[`Linux-RelWithDebInfo-29b20829.tar.xz`]
+| 🕵️‍♂️ RelWithDebInfo
+| 🕵️‍♂️ RelWithDebInfo
+| ✅
+| ✅
+| N/A
+
+| 🐧 https://mrdox.com/llvm+clang/Linux-DebWithOpt-29b20829.tar.xz[`Linux-DebWithOpt-29b20829.tar.xz`]
+| 🔬 DebWithOpt
+| 🐞 Debug
+| ✅
+| ✅
+| N/A
+|===
+
+IMPORTANT: The Linux binaries are built on Ubuntu 22.04 and may not work on other distributions.
+
+You can download the binaries and uncompress them in the `./third-party/llvm+clang` directory we created in the previous step.
+
+LLVM binaries are provided in a number of preset configurations.
+Here is a brief description of each preset:
+
+- `Release`: this is the preset users will typically use.
+It is optimized for speed and does not include debug information.
+- `Debug`: this is a preset developers can use.
+It includes debug information and no optimizations.
+However, using a `Debug` build of LLVM to debug MrDocs might be too slow.
+In this case, you can link MrDocs with `RelWithDebInfo` or `DebWithOpt`.
+- `RelWithDebInfo`: this is a release build with debug information.
+It is optimized for speed and includes debug information.
+However, if you are working with Windows+MSVC, this preset has a `Release` build type at the MSVC level.
+This means you can have conflicts with MrDocs in `Debug` mode because of LLVM setting flags such as the `_ITERATOR_DEBUG_LEVEL` and `/MDd`.
+In this case, you can use `DebWithOpt` instead to avoid the conflict and subsequent workarounds.
+- `DebWithOpt`: this is a debug build with optimizations.
+It includes all the default Debug flags for LLVM, it's optimized for speed, includes debug information, and causes no conflicts with MrDocs in `Debug` mode.
+
+If you chose to use the provided binaries instead of building LLVM from source, you can skip to the <<duktape>> section.
+
+**Download**:
 
 Alternatively, if building LLVM from source, you can clone the project from the official repository:
 
 [source,bash]
 ----
 git clone https://github.com/llvm/llvm-project.git
 cd llvm-project
-git checkout 731264b0c2af7aa46bd39625202a99e06cfccff9
+git checkout 29b20829cc6ce3e6d9c3809164994c1659e0da56
+cd llvm
 ----
 
+**Configure**:
+
+There are two ways to configure LLVM: using <<llvm-configure-cmd-line, CMake directly>> or using <<llvm-configure-presets, CMake presets>>.
```

Comment:
Yes. I agree. 

I just left it there because I didn't want to mess with or trigger arguments about older stuff in case anyone is emotionally attached to it.

I'll remove it.

Reply:
On second thought, someone using a slightly older version of CMake could need the command line. I don't know. On third thought, that's kind of unlikely because of the compilers we require. Now I'm wondering if it's best to be safe than sorry.